app/models/manageiq/providers/lenovo/physical_infra_manager/refresh_parser.rb

```diff
@@ -76,17 +76,28 @@ def parse_physical_server(node)
         :model                  => node.model,
         :serial_number          => node.serialNumber,
         :field_replaceable_unit => node.FRU,
-        :computer_system        => {
-          :hardware             => {
-            :networks  => [],
-            :firmwares => [] # Filled in later conditionally on what's available
-          }
-        }
+        # Filled in later conditionally on what's available
+        :computer_system        => {:hardware => {:networks => [], :firmwares => []}}
       }
       new_result[:computer_system][:hardware] = get_hardwares(node)
+      save_host_relationship(node)
       return node.uuid, new_result
     end
 
+    def save_host_relationship(node)
```

Comment:
@juliancheal I don't figured out how to send a save host to MIQ core from here. I mean, the ID of relationship was saved in Physical server, as you guys defined, so, once we found the Host ID and the Physical server ID which matches considering the LXCA association(serial_number and service_tag equals), how to do that in Core?